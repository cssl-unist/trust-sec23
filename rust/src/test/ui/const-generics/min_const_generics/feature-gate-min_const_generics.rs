fn test<const N: usize>() {}
//~^ ERROR const generics are unstable

fn main() {}
