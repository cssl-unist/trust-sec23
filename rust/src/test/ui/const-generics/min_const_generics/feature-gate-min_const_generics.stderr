error[E0658]: const generics are unstable
  --> $DIR/feature-gate-min_const_generics.rs:1:15
   |
LL | fn test<const N: usize>() {}
   |               ^
   |
   = note: see issue #74878 <https://github.com/rust-lang/rust/issues/74878> for more information
   = help: add `#![feature(min_const_generics)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
